homeassistant/components/bluecurrent/const.py

```diff
@@ -0,0 +1,190 @@
+"""Constants for the BlueCurrent integration."""
+import logging
+
+from homeassistant.components.sensor import SensorEntityDescription
+
+DOMAIN = "bluecurrent"
+
+LOGGER = logging.getLogger(__package__)
+
+PLATFORMS = ["sensor"]
+CHARGE_POINTS = "CHARGE_POINTS"
+DATA = "data"
+DELAY_1 = 1
+DELAY_2 = 20
```

Comment:
Would be nice to document what delays these are, either by comment or by a better name.

Reply:
Renamed to `SMALL_DELAY` and `LARGE_DELAY`.